fronts-client/src/bundles/chefsBundle.ts

```diff
@@ -1,28 +1,71 @@
 import createAsyncResourceBundle from '../lib/createAsyncResourceBundle';
 import { Chef } from '../types/Chef';
-import chefOttolenghi from './fixtures/chef-ottolenghi.json';
-import chefStein from './fixtures/chef-stein.json';
-import chefCloake from './fixtures/chef-cloake.json';
 import { selectCard } from 'selectors/shared';
 import { State } from 'types/State';
 import { createSelector } from 'reselect';
 import { stripHtml } from 'util/sanitizeHTML';
+import { ThunkResult } from 'types/Store';
+import { previewCapi, liveCapi } from 'services/capiQuery';
+import { Tag } from '../types/Capi';
 
-const sanitizeChef = (chef: Chef) => ({
-  ...chef,
-  bio: stripHtml(chef.bio ?? ''),
+const sanitizeTag = (tag: Tag) => ({
+  ...tag,
+  bio: stripHtml(tag.bio ?? ''),
 });
 
 const bundle = createAsyncResourceBundle<Chef>('chefs', {
   indexById: true,
-  initialData: {
-    // Add stub data in the absence of proper search data.
-    [chefOttolenghi.id]: sanitizeChef(chefOttolenghi),
-    [chefStein.id]: sanitizeChef(chefStein),
-    [chefCloake.id]: sanitizeChef(chefCloake),
-  },
+  selectLocalState: (state) => state.chefs,
 });
 
+const fetchResourceOrResults = async (
+  capiService: typeof liveCapi,
+  params: object,
+  isResource: boolean,
+  fetchFromPreview: boolean = false
+) => {
+  const capiEndpoint = capiService.chefs;
+  const { response } = await capiEndpoint(params);
+
+  return {
+    results: response.results,
+    pagination: {
+      totalPages: response.pages,
+      currentPage: response.currentPage,
+      pageSize: response.pageSize,
+    },
+  };
+};
+
+export const createFetch =
```

Comment:
I'd recommend refactoring this to remove the additional wrapping function – as I think we'll always want tags from live CAPI, and we'll always want to use bundle.actions.

So this function might be renamed `fetchChefs`, and `fetchLive` below would no longer be needed.